bower.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "customapps-medallia-library-FileSaver.js",
+  "version": "0.0.1",
```

Comment:
Oh, I got the version number from above from https://github.com/eligrey/FileSaver.js/blob/master/FileSaver.js#L3 (line 3).  Maybe not quite a "real" version?